src/main/java/Price.java

```diff
@@ -19,14 +19,14 @@ private void validateLottoPriceUnit(String value) {
             return;
         }
 
-        throw new IllegalArgumentException("Invalid lotto price value: " + value);
+        throw new IllegalArgumentException("로또 단위는 1000원 입니다. 1000원 단위로 입력해주세요 입력된 (" + value + ")는 1000원 단위의 수가 아닙니다.");
```

Comment:
String의 format 메서드를 사용하면 로또의 가격이 변경됐을 때
수정할 부분이 줄어들 수 있어서 추천드립니다! 😉 
```suggestion
        throw new IllegalArgumentException(String.format("로또 단위는 %d원 입니다. %d원 단위로 입력해주세요 입력된 (%d)는 %d원 단위의 수가 아닙니다.", LOTTO_PRICE, LOTTO_PRICE, value, LOTTO_PRICE));
```